Dockerfile

```diff
@@ -1,16 +1,25 @@
-FROM python:3.7-alpine AS build
+FROM python:3.7-alpine AS base
+
+# ensure latest updates are installed (security)
+RUN apk upgrade --no-cache
```

Comment:
This is against the docker best practices. Never upgrade packages in the image itself. The base should be upgraded 

> Avoid RUN apt-get upgrade and dist-upgrade, as many of the “essential” packages from the parent images cannot upgrade inside an unprivileged container

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/

Reply:
But "I" have no control over the base image, I have no control when they will update their packages. To incorporate a security update docker:alpine needs to be updated, after this docker:python needs to be update, and finally docker:python-gitlab has to be updated and maybe any custom company specific image based on python-gitlab has to be updated too.

But I want this fix for the pecl lib now which is used by some version-parser-module to extract version info from tags which holds some malicious formatted content. (Just an example)

So, i do not agree to this best-practice. :-)

Reply:
I do agree to follow best-practice and we shouldn't maintain the base image of python ourselves.

> But "I" have no control over the base image

These changes should rather go here: https://github.com/docker-library/python